### PR TITLE
Feature/deployable docker build

### PR DESCRIPTION
**Pull request by stutily**
Extra files for building deployable version for k8s which locks the code at a particular version and imports the codebase into the container 

There should be no impact from this to the existing master branch as it just adds a few new files

### CODE REVIEW

File: docker-build/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+(note - the build expects a version file as build-version.txt in the root which gets added by
+concourse - this will need manually adding for the local build. it's just informational so touch
+the file there)
+
+*As of current docker version (19.03) the DOCKER_BUILDKIT is used to allow per build docker ignore files 
+
+The overall process is as follows 
+
+- Something runs composer (either a developer locally or else a concourse pipeline) for PHP installs
+(* and then ny npm installs - TODO) to get the checked out version fully setup, so as this build can create an
```
Comment (joesb) [inline docker-build/README.md]: Right, yes, deets on the necessary `npm install` process noted. Will pass that across to you.

File: web/sites/default/settings.prod-eks.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+$settings['s3fs.bucket'] = getenv('AWS_BUCKET_NAME');
+$settings['s3fs.region'] = getenv('AWS_REGION');
+$config['s3fs.settings']['bucket'] = getenv('AWS_BUCKET_NAME');
+$settings['s3fs.use_s3_for_public'] = TRUE;
+
+// Sendgrid integration API key
+$config['sendgrid_integration.settings']['apikey'] = getenv('CONNECT_SENDGRID_API_KEY');
+
+// Ensure the devel config environment is off
+$config['config_split.config_split.devel']['status'] = FALSE;
```
Comment (joesb) [inline web/sites/default/settings.prod-eks.php]: Could probably do with a blank line at the end of this file — PHP convention

Comment (stutily) [reply web/sites/default/settings.prod-eks.php]: done in 6877cda77c60809b2c692e39533b84f738ba0db7

**Comment by joesb** [comment]
Ace, thanks!

**Pull request merged**